java/arcs/sdk/wasm/WasmHandle.kt

```diff
@@ -34,4 +34,6 @@ abstract class WasmHandleEvents<T>(
     fun onUpdate(action: (T) -> Unit) {
         onUpdateActions.add(action)
     }
+
+    abstract fun getContent(): T
```

Comment:
I don't think we should be adding extra user visible methods to the handle class. We don't want users to invoke getContent(), and they will we put it here.

I would prefer if we introduced a generic type on the Handle superclass, so that we had:

Handle<T>
Singleton<T : Entity> : Handle<T?>
Collection<T : Entity> : Handle<Set<Entity>>

and then we  can have a method:

<A, B, C> combine(a: Handle<A>, b: Handle<B>, c:  Handle<C>)